src/server/standalone.js

```diff
@@ -194,9 +195,19 @@ function StandAloneServer(gmeConfig) {
             }
         });
 
-        __httpServer.listen(gmeConfig.server.port, function () {
+        // something link
+        // https://github.com/rubenv/node-autoquit
+        // should be considered to shut the service down if it
+        // is not getting much use.
+
+        // https://github.com/herzi/systemd-socket
+        // https://nodejs.org/api/http.html#http_server_listen_handle_callback
+        __httpServer.listen(systemdSocket(0) || gmeConfig.server.port, function () {
```

Comment:
Are you suggesting moving the systemd code to one of the 'config/config.*.js' files?
Yes, that would probably work fine.